synapse/storage/util/sequence.py

```diff
@@ -39,6 +39,12 @@ def get_next_id_txn(self, txn: Cursor) -> int:
         txn.execute("SELECT nextval(?)", (self._sequence_name,))
         return txn.fetchone()[0]
 
+    def get_next_mult_txn(self, txn: Cursor, n: int) -> List[int]:
+        txn.execute(
+            "SELECT nextval(?) FROM generate_series(1, ?)", (self._sequence_name, n)
+        )
+        return [i for i, in txn]
```

Comment:
Might be worth putting parentheses around `i,`? (Unless black reformatted them out) The comma looks like a typo otherwise.